supporting_scripts/analysis-of-endpoint-connections/eslint.config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "jsx": true
+        }
+    },
+    "rules": {},
+    "settings": {
+        "react": {
+            "version": "detect"
+        }
+    }
```

Comment:
Just curious: why do we need to specify a React version?

Reply:
We don't 🙈